src/utils/signaling.js

```diff
@@ -1499,6 +1500,26 @@ Signaling.Standalone.prototype.processRoomParticipantsEvent = function(data) {
 				console.error('Unknown room participant event', data)
 			}
 		} else {
+			// Convert participant data in federated conversations
+			if (this.settings.federation?.nextcloudServer) {
+				for (let i = 0; i < data.event.update.users.length; i++) {
+					const user = data.event.update.users[i]
+					if (user.actorType === ATTENDEE.ACTOR_TYPE.USERS) {
+						user.actorType = ATTENDEE.ACTOR_TYPE.FEDERATED_USERS
+						user.actorId = user.actorId + '@' + this.settings.federation.nextcloudServer
+					} else if (user.actorType === ATTENDEE.ACTOR_TYPE.FEDERATED_USERS
+							&& (user.actorId.endsWith('@' + getBaseUrl())
+								|| user.actorId.endsWith('@' + getBaseUrl().replace('https://', ''))
+							)) {
+						let indexOfDomain = user.actorId.indexOf('@' + getBaseUrl())
```

Comment:
is there something like `last index of` 
Unlikely but could happen :see_no_evil:

Reply:
Correct, but no longer needed as commit was dropped.